controllers/archive_controller.go

```diff
@@ -39,7 +40,11 @@ func (r *ArchiveReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
-	config := job.NewConfig(ctx, r.Client, log, archive, r.Scheme)
+	repository := cfg.GetGlobalRepository()
```

Comment:
I'm not sure how much sense these snippets make. Because the `Backend.String()` basically already does what `GetGlobalRepository()` does but will also consider if there are any overrides within the `Backend` instance. See https://github.com/vshn/k8up/blob/master/api/v1alpha1/backend.go#L87

Reply:
the point is: if Backend is nil, that won't work.

Reply:
Ah right, yes then it makes sense.